nuitka/tools/release/Documentation.py

```diff
@@ -28,68 +28,65 @@
 from nuitka.utils.FileOperations import getFileList
 
 
+def which(prog):
+    """
+    cross version program detection.
+
+    Whenever Nuitka decides to drop Python versions older than 3.3
+    the code in the else block can be removed.
+    """
+    def real(f):
+        def wrap(*args):
+            cmd = ""
+            if sys.version_info > (3, 3):
+                import shutil
+                cmd = shutil.which(prog)
+            else:
+                _, cmdname = os.path.split(prog)
+
+                for path in os.environ["PATH"].split(os.pathsep):
+                    cmd_ = os.path.join(path, cmdname)
+                    if os.path.isfile(cmd_) and os.access(cmd_, os.X_OK):
+                        cmd = cmd_
+            if cmd:
+                f(*args)
+            else:
+                print("Could not find such program %s" % prog)
```

Comment:
That is better as `sys.exit` with a string argument. Errors in stdout are not good.

Reply:
Correct. I changed. this.